pkgs/top-level/all-packages.nix

```diff
@@ -2571,8 +2571,9 @@ with pkgs;
 
   microcom = callPackage ../applications/terminal-emulators/microcom { };
 
-  mlterm = darwin.apple_sdk_11_0.callPackage ../applications/terminal-emulators/mlterm {
-    inherit (darwin.apple_sdk_11_0.frameworks) Cocoa;
+  mlterm = callPackage ../applications/terminal-emulators/mlterm { };
```

Comment:
```suggestion
  mlterm = darwin.apple_sdk_11_0.callPackage ../applications/terminal-emulators/mlterm { };
```

Reply:
What is wrong here? Why do we need this `callPackage`?

Reply:
Because we need a newer apple SDK for x86_64-darwin.

Reply:
In other words, a dreaded magic that ~~we~~ I think is too dirty. 
The typical case of [https://stackoverflow.com/a/778275](temporary-my-ass).

More about it at https://github.com/NixOS/nixpkgs/issues/204303